src/components/Records/tests/Pagination.spec.js

```diff
@@ -55,44 +62,28 @@ describe("Pagination.vue", () => {
         expect(anotherWrapper2.vm.currentPage).toBe(120);
     });
 
-    it('can react to changes in the URL query and/or name', () => {
-        wrapper.vm.$route.name = "Search";
-        wrapper.vm.$route.query =  {};
-        expect(wrapper.vm.currentPage).toBe(1);
-    });
-
-    it('Has a paginate() method that sets the current page in the URL query', () => {
-        $route.name="Standards";
-        $route.query.page=12;
-
-        wrapper.vm.currentPage = 2;
-        wrapper.vm.$route.query.page = "3";
-        wrapper.vm.paginate(3);
-        expect(wrapper.vm.currentPage).toBe(3);
-        wrapper.vm.paginate(3);
-        expect(wrapper.vm.currentPage).toBe(3);
-        expect(wrapper.vm.$route.name).toBe("Standards");
-        expect(wrapper.vm.$route.query.page).toBe("3");
-    });
 
     it('Has a first() method that always redirect to the first page', () => {
 
+
         wrapper.vm.currentPage = 10;
         wrapper.vm.$route.query.page = "1";
 
+        wrapper.vm.allowPaginate = true;
         wrapper.vm.$router.push = push;
         wrapper.vm.first();
         wrapper.vm.$route.query.page = "1";
-
         expect(wrapper.vm.currentPage).toBe(1);
         expect(wrapper.vm.$route.query.page).toBe("1");
```

Comment:
This is not very usefull since you go wrapper.vm.$router.query.page = "1" two lines above.
Replace it with:          expect($router.push).toHaveBeenCalledTimes(1);
This way we tests that $router.push has been called the first time, and not the second (see line 93).

Reply:
its done